leaderboards/leaderboard_challenges.tex

```diff
@@ -0,0 +1,160 @@
+    \label{fig:passing_leaderboard}
+\end{figure}
+\subsubsection{Challenge Execution}
+In each attempt, participating teams must start the robot behaviour of their competing robot via a single chest button
+press. Teams will then have \qty{\MaxTimePassingLeaderboard}{\sec} to move the ball from the starting
+position to the target area while completing as many passes as possible. The centre
+1.5 metres of the field (Centre Cricle Diameter) is an exclusion zone, no robots are allowed within this area,
+if a robot enters this area, or the ball comes to a complete stop
+within this area, the run is ended immediately. If the ball is kicked off the field,
+it is placed back on the touchline where it crossed. Each run is considered complete
+once a robot has posession of the ball within the target area.
+
+\subsubsection{Scoring}
+1 point is awarded every time the ball crosses the exlusion zone, with an additional point
+awarded if the ball touches the recieving robot before it stops rolling.
+
+No points are counted if the time runs out before a robot has posession of the ball
+within the target area, or if the run ends prematurely as per the above rules.
+
+The highest score of the three runs will be recorded in the leaderboard.
+
+\subsection{Ball Control Leaderboard}
+This leaderboard will measure how well a NAO can keep control of a ball, while moving it
+to a target.
+
+\subsubsection{Setup}
+This leaderboard challenge will take place on one half of a standard SPL field, and requires 1 active robot, 4 inactive robots and 1 SPL Soccer Ball.
+Inactive robots will act as obstacles for the attempt and will be placed as follows (illustrated in \cref{fig:walk_leaderboard}):
+\begin{itemize}
+    \item On the penalty mark
+    \item On both corners of the goal area that are not touching the field boundary
+    \item On the corner of the goal area closest to the competing robot
+\end{itemize}
+The ball is placed on the touchline in line with the penalty mark, with the competing
+robot placed around 50cm behind it.
+The end goal is the touchline on the other side of the field.
+
+\begin{figure}[t]
+    \centerline{\includegraphics[width=\columnwidth]{figs/control_leaderboard.pdf}}
+    \caption{Ball Control Leaderboard: Robot completing the attempt is shown in blue, target is shown in green, obstacles are shown in red and the ball is shown in yellow. An example path is shown in magenta}
+    \label{fig:ball_control_leaderboard}
+\end{figure}
+
+\subsubsection{Challenge Execution}
+In each attempt, participating teams must start the robot behaviour of their competing robot via a single chest button
+press. Participating teams will be given 3 attempts to dribble the ball around the obstacles and across
+the goal touchline in as fast a time as possible.
+
+Anytime the ball moves more than 75cm away from the competing robot or the ball comes in contact with
```

Comment:
Will this be announced and added when it occurs, or a review of footage be warranted afterwards and then applied?

Reply:
Yes good idea, announcing at the time makes sense